template/.npmignore.temp

```diff
@@ -1,16 +1,23 @@
 # standard ignore
+*config*
+*conf*
+
+.github/
 .gitignore
+.eslintrc
 .stylelintrc
-karma.conf.js
-rollup.config.js
-babel.config.js
-.github/
-index.html
+
+coverage/
+docs/
```

Comment:
Whats the reason to stop including the docs with the npm install process?

Reply:
Honestly, I have sporadically but truly not that often. Seems to me that including them is trivial and has nearly no cost to the user.